src/app/shared/components/template/components/task-card/task-card.component.ts

```diff
@@ -101,6 +93,7 @@ export class TmplTaskCardComponent extends TemplateBaseComponent implements OnIn
   completedIcon: string;
   inProgressIcon: string;
   isButton: boolean;
+  variant: string;
```

Comment:
nit(blocking)
I think moving forward really important we define the variants in the code so that we are aware of what is available. Although the authored variant could be a list container multiple, e.g. `portrait background-secondary` I think for now still fine to just add type definitions as if single string `variant: portrait | background-secondary`